scss/_base_lists.scss

```diff
@@ -1,27 +1,24 @@
 @mixin vf-lists {
+  
   ol,
   ul {
-    margin-left: 20px;
-    margin-bottom: 20px;
-  }
+    margin-left: 1rem;
+    margin-bottom: 1rem;
+    padding-left: 1rem;
 
-  ol ol,
-  ul ul,
-  ol ul,
-  ul ol {
-    margin-bottom: 0;
-  }
+    ul,
+    ol, {
+      margin-bottom: 0;
```

Comment:
I think we want lists to have the same bottom margin as p tags. Otherwise they will look bad in text.